Zune.Net.Catalog.Image/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
     "ZuneNetContext": {
-        "ConnectionString": "mongodb://localhost:27017",
+        "ConnectionString": "mongodb://mongo:27017",
```

Comment:
In docker, you can expose things like connection strings to a running container as an environment variables. I'm gonna probably revert this to localhost and define the runtime connection strings in `docker-compose.yml`